README.md

```diff
@@ -226,6 +226,37 @@ namespace/name is used as the OAEP input parameter, ensuring that the
 The generated `Secret` is marked as "owned" by the `SealedSecret` and
 will be garbage collected if the `SealedSecret` is deleted.
 
+## Developing
+To be able to develop on this project, you need to have the following tools installed:
+* make
+* [Ginkgo](https://onsi.github.io/ginkgo/)
+* [Minikuke](https://github.com/kubernetes/minikube)
+* [kubecfg](https://github.com/ksonnet/kubecfg)
+* Go
+
+To build the `kubeseal` and controller binaries, run:
+```bash
+$ make
+```
+
+To run the unit tests:
+```bash
+$ make test
+```
+
+To run the integration tests:
+* Start Minikube
+* Build the controller for Linux, so that it can be run within a Docker image - edit the Makefile to add 
+`GOOS=linux GOARCH=amd64` to `%-static`, and then run `make controller.yaml sealedsecret-crd.yaml`
+* Alter `controller.yaml` so that `imagePullPolicy: Never`, to ensure that the image you've just built will be
```

Comment:
Fwiw, I use `make CONTROLLER_IMAGE=sealed-secrets-controller:test` so I avoid the `:latest` pull policy.  That probably means we should just change the `CONTROLLER_IMAGE` default in `Makefile` to also avoid `:latest` ...